cpp/src/detail/utility_wrappers_32.cu

```diff
@@ -81,13 +81,18 @@ template void transform_increment_ints(raft::device_span<int32_t> values,
                                        int32_t value,
                                        rmm::cuda_stream_view const& stream_view);
 
-template void transform_cast_ints(raft::device_span<int32_t> new_vertices,
-                                  raft::device_span<int32_t const> old_vertices,
-                                  rmm::cuda_stream_view const& stream_view);
-
-template void transform_cast_ints(raft::device_span<int32_t> new_vertices,
-                                  raft::device_span<int64_t const> old_vertices,
-                                  rmm::cuda_stream_view const& stream_view);
+template void copy_or_transform(raft::device_span<int32_t> output,
```

Comment:
These would move to `graph_helper_sg.cu`.  Won't need a copy in _mg file, only one is necessary.